src/Models/HydrostaticFreeSurfaceModels/compute_hydrostatic_free_surface_tendencies.jl

```diff
@@ -134,7 +134,7 @@ end
 ##### Boundary condributions to hydrostatic free surface model
 #####
 
-function apply_flux_bcs!(Gcⁿ, c, arch, args)
+function apply_flux_bcs!(Gcⁿ, c, arch, args::Vararg{Any, N}) where {N}
```

Comment:
Ok @jlk9 this is specifically what you're referring to? Let's show there's no slowdown and then we can merge. (already from the CI results it looks like we might be good)